docs/_packages/web_api.md

```diff
@@ -448,15 +448,77 @@ retrying the API call. If you'd like to opt out of that behavior, set the `rejec
 ---
 
 ### Upload a file
+As of @slack/web-api v2.8.0, we have introduced a modified way to upload files. 
```

Comment:
```suggestion
As of @slack/web-api v6.8.0, we have introduced a modified way to upload files. 
```

Reply:
Ooh, thanks for the catch! I've fixed it.